src/components/molecules/Chatbox/Chatbox.tsx

```diff
@@ -59,29 +74,68 @@ export const Chatbox: React.FC<ChatboxProps> = ({
     [messages, deleteMessage]
   );
 
+  const dropdownOptions = useMemo(
+    () =>
+      OPTIONS.map((option) => (
+        <Dropdown.Item
+          key={option.name}
+          onClick={() => setActiveOption(option)}
+        >
+          {option.name}
+          <FontAwesomeIcon icon={option.icon} />
+        </Dropdown.Item>
+      )),
+    []
+  );
+
+  const showPoll = useMemo(() => activeOption?.name !== OPTIONS[0].name, [
+    activeOption,
+  ]);
+
+  const handleSubmit = useCallback((data) => console.log(data), []);
+
+  const renderForms = useMemo(() => {
+    switch (activeOption?.name) {
+      case OPTIONS[0].name:
+        return <PollBox onSubmit={handleSubmit} />;
+
+      default:
+        return <ChatMessageBox sendMessage={sendMessage} />;
+    }
+  }, [activeOption, sendMessage, handleSubmit]);
+
   return (
     <div className="chatbox">
-      <div className="chatbox__messages">{renderedMessages}</div>
-      <form className="chatbox__form" onSubmit={onSubmit}>
-        <input
-          className="chatbox__input"
-          ref={register({ required: true })}
-          name="message"
-          placeholder="Write your message..."
-          autoComplete="off"
-        ></input>
-        <button
-          className="chatbox__submit-button"
-          type="submit"
-          disabled={!chatValue || isSendingMessage}
-        >
-          <FontAwesomeIcon
-            icon={faPaperPlane}
-            className="chatbox__submit-button-icon"
-            size="lg"
-          />
-        </button>
-      </form>
+      <div className="chatbox__messages">
+        {
+          renderedMessages
+          // <ChatPoll
+          //   pollData={pollData}
+          //   deletePoll={console.log}
+          // />
+        }
+      </div>
+      <div className="chatbox__container">
+        {showPoll ? (
+          <DropdownButton
+            id="options-dropdown"
+            title="Options"
+            className="chatbox__dropdown"
+            variant="link"
+            drop="up"
+          >
+            {dropdownOptions}
+          </DropdownButton>
+        ) : (
+          <div
+            className="chatbox__cancel-poll"
+            onClick={() => setActiveOption(null)}
+          >
+            Cancel Poll
+          </div>
+        )}
```

Comment:
Is the logic here right? It looks like it's going to render the options menu when `showPoll` is true, and 'cancel poll' otherwise; yet I would have thought we want the opposite of that?

Reply:
I wonder if this section could also be refactored to use `select`/`case` or similar to make it cleaner once @mike-lvov's suggestion in https://github.com/sparkletown/sparkle/pull/1343/files?diff=unified&w=1#r622457060 is done?

Reply:
fixed in b1c8ac0